### ISSUE TITLE
[Bug]: Error: Cannot find module 'core-js/library/fn/object/define-property'

### ISSUE THREAD

**Issue by weichtg3** [babel/babel#14508]
### 💻

- [ ] Would you like to work on a fix?

### How are you using Babel?

Other (Next.js, Gatsby, vue-cli, ...)

### Input code

```js
CircleCI build started failing within the past hour and seems to be related to v7.17.10.

"@babel/runtime-corejs2": "^7.13.10",  got resolved to 7.17.10 and our unit test step started failing:

> mkdir -p conf && touch conf/server.key && touch conf/server.crt

Error: Cannot find module 'core-js/library/fn/object/define-property'
Require stack:
- /home/node/project/node_modules/@babel/runtime-corejs2/core-js/object/define-property.js
- /home/node/project/test/acl_helper.spec.js
- ...
- ...

When manually installing 7.17.9 the error went away and our unit test script ran fine (npm).
```

### Configuration file name

package.json

### Configuration

```json
  "babel": {
    "presets": [
      "@babel/preset-env"
    ],
    "plugins": [
      [
        "istanbul"
      ],
      [
        "@babel/plugin-proposal-object-rest-spread"
      ],
      [
        "@babel/plugin-transform-runtime",
        {
          "corejs": 2,
          "regenerator": true
        }
      ]
    ]
  },
```

### Current and expected behavior

Current behavior is the error noted above which prevents our CircleCI jobs from successfully completing and deploying.  Expected behavior is for things to work like they did prior to this version.

### Environment

- CircleCI
- node v16.14.0
- npm v6.14.6

**Comment by JLHwung** [comment]
This is a regression introduced at https://github.com/babel/babel/pull/14474, fixes are on the way.

**Comment by JLHwung** [comment]
Fixed in `@babel/runtime-corejs2` 7.17.11.
